File: java-example/src/main/java/com/pushbots/java_example/MyApplication.java
package com.pushbots.java_example;

import android.app.Application;

import com.pushbots.push.Pushbots;

public class MyApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();

        // Initialize PushBots
        Pushbots.sharedInstance().init(this);
    }
}

File: pushbots-plugin-example/src/main/java/com/gradle/plugin/pushbots_plugin_example/MyApplication.kt
package com.gradle.plugin.pushbots_plugin_example

import android.app.Application
import com.pushbots.push.Pushbots

class MyApplication : Application() {

    override fun onCreate() {
        super.onCreate()

        // Initialize PushBots
        Pushbots.sharedInstance().init(this)
    }
}